scripts/chwd

```diff
@@ -124,17 +128,20 @@ local function exec_hook(hook)
         return
     end
 
-    if hook ~= "" then
-        local handle, errmsg = io.popen(("/bin/bash -c '%s'"):format(hook), "r")
+    if hook == "" then
+        return
+    end
 
-        if handle then
-            local output = handle:read('*all')
-            handle:close()
-            return output
-        else
-            print(("ERROR: Unkown shell invocation error for %s hook: %s"):format(hook, errmsg))
-        end
+    local handle, errmsg = io.popen(hook, "r")
+
+    if not handle then
+        printf("ERROR: Unkown shell invocation error for %s hook: %s", hook, errmsg)
+        return
     end
+
+    local output = handle:read("*a")
+    handle:close()
```

Comment:
maybe should check if hook failed or not. close will return exit code

Reply:
Done, plus fixing the output of hook name.